index.bs

```diff
@@ -10467,177 +10467,195 @@ asynchronous sections can't be reordered.
 <h3 id="rendering-loop">
 Rendering an audio graph</h3>
 
-Rendering an audio graph is done in blocks of 128 samples-frames. A
+Rendering an audio graph is done by processing blocks of 128 samples-frames. A
 block of 128 samples-frames is called a <dfn>render quantum</dfn>.
 
-Operations that happen <dfn>atomically</dfn> on a
-given thread can only be executed when no other [=Atomically|atomic=]
-operation is running on another thread.
+Operations that happen <dfn>atomically</dfn> on a given thread can only be
+executed when no other [=Atomically|atomic=] operation is running on another
+thread.
 
-The algorithm for rendering a block of audio from an
-{{AudioContext}} <em>G</em> with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is as
-follows.
+The algorithm for rendering a block of audio from an {{AudioContext}} <em>G</em>
+with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is
+comprised of 3 steps:
+	1. Process the control message queue.
+	2. Process a render quantum.
+	3. Perform a microtask check point.
```

Comment:
Maybe just use "1" as the item number so it never has to be renumbered if something is added to or removed from the list?

Reply:
It's only 3 steps and we should be explicit about it. Numbering all of them into `1` does not really help anything but the editorial convenience.

Reply:
Except when it churns numbers later on.  But you can leave it as is.